skrf/network.py

```diff
@@ -374,6 +378,7 @@ def __init__(self, file=None, name=None, comments=None, f_unit=None, **kwargs):
 
         self.name = name
         self.comments = comments
+        self.s_definition = s_definition
```

Comment:
we might need this in Network.copy() as well.

Reply:
OK